### PR TITLE
Consider isGxP project flag in SSDS templates.

### PR DESCRIPTION
**Pull request by valituguran**

### CODE REVIEW

File: templates/SSDS-1.html.tmpl [hunk 1]
```diff
@@ -142,7 +142,7 @@
...
                     <br/>
-                    A person who creates a PR gets attributed as its author and nominates at least one reviewer. Once mandatory changes requested by the reviewers have been incorporated by the contributors, reviewers can approve the PR. An approval permits that the approved contribution can be merged from a temporary source branch into a longer living target branch inside the repository. A single person must not assume the roles of contributor and reviewer on the same PR (no self-approvals).<br/>
+                    A person who creates a PR gets attributed as its author and nominates at least one reviewer. Once mandatory changes requested by the reviewers have been incorporated by the contributors, reviewers can approve the PR. An approval permits that the approved contribution can be merged from a temporary source branch into a longer living target branch inside the repository.{{#if data.sections.sec2s3.isGxpProject}} A single person must not assume the roles of contributor and reviewer on the same PR (no self-approvals).{{/if}}<br/>
```
Comment (metmajer) [inline templates/SSDS-1.html.tmpl:145]: @valituguran it appears that `isGxpProject is a property of documentation section 2.3 (via `data.sections.sec2s3`). However, `isGxPProject` should be a global property. Therefore I suggest you use `data.isGxPProject` instead.

**Review by metmajer — CHANGES_REQUESTED** [review]
Hello @valituguran, please look into reworking the data structure, thanks!

File: templates/SSDS-1.html.tmpl (hunk 1 above)
Comment (valituguran) [reply templates/SSDS-1.html.tmpl]: @metmajer the changes have been done.

**Review by metmajer — APPROVED** [review]
Well done, @valituguran. Please squash and merge into a single commit. Thanks!

**Pull request merged**